### ISSUE TITLE
GLIBC_2.28 missing on docker:latest and v5.21.0

### ISSUE THREAD

**Issue by stvno** [Project-OSRM/osrm-backend#5324]
I'm trying to run the osrm-routed on the latest docker images but I get an error:
```
$ docker run -ti -p 5000:5000 -v $(pwd)/data:/data osrm/osrm-backend:latest osrm-routed  /data/netherlands-latest.osrm

osrm-routed: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by osrm-routed)
```

It happens both with `osrm/osrm-backend:latest` and `osrm/osrm-backend:v5.21.0` it works however with `osrm/osrm-backend:v5.20.0`

**Comment by danpat** [comment]
@stvno This should get fixed by https://github.com/Project-OSRM/osrm-backend/pull/5323, but I haven't had a chance to merge that yet.

**Comment by rajasaur** [comment]
@danpat : Can you publish the latest image with this fixed, or are there other things blocking a release.

**Comment by danpat** [comment]
I finally got some time to dig into this - looks like there have been upstream changes in Debian `buster` that have fixed this already - doing:

```
docker build -f docker/Dockerfile .
```

to build OSRM locally now *does not* complain about the missing `GLIBC_2.28` symbol.

So I suspect the root cause here was some instability in dependencies in `buster`.

I've closed #5323 and merged https://github.com/Project-OSRM/osrm-backend/pull/5338 instead - by downgrading the base image to `stretch-slim`, we should have a much more stable set of dependencies.
